bridge/keptn-bridge/.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Are the files in the .idea directory meant to be added to the repo?

Reply:
Added by accident. I just removed them again.